requirements.txt: add missing dependencies

claviola opened the pull request.

tkluck commented:
Thanks for the pull request! Looking at the history for `requirements.txt`, looks like we intentionally removed those dependencies to enable installation on macOS, without really providing another solution.

We should probably move these requirements to `requirements-linux.txt` or something and update the `make env` target accordingly?

claviola commented:
I've updated it by restricting txdbus and systemd to Linux and removed pacparser entirely, as it's [docs](https://github.com/manugarg/pacparser/blob/master/INSTALL) recommend manual building. It is available on PyPI, however: https://pypi.org/project/pacparser/
